src/image/filter/subtractImage.js

```diff
@@ -12,7 +12,7 @@ import Image from '../Image';
  * @return {Image}
  */
 export default function substractImage(otherImage, options = {}) {
-    let {bitDepth = this.bitDepth, channels} = options;
+    let {bitDepth = this.bitDepth, channels, absolute = false} = options;
     this.checkProcessable('substractImage', {
         bitDepth: [8, 16]
     });
```

Comment:
alpha must equal 0. Would not make sense to subtract alpha. Or maybe we should skip the alpha channel if it exists ?